content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+single YAML file from a directory of YAML files and combines them as an 
+[OCI container image](https://opencontainers.org/).  
+
+The CLI applies the required annotations and values to meet the 
+[Crossplane XPKG specification](https://github.com/crossplane/crossplane/blob/master/contributing/specifications/xpkg.md).
+
+The `crossplane` CLI supports building 
+[configuration]({{< ref "../concepts/packages" >}}),
+[function]({{<ref "../concepts/composition-functions">}}) and 
+[provider]({{<ref "../concepts/providers" >}}) package types. 
+
+
+#### Flags
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                            | Description                    |
+| ------------ | -------------                        | ------------------------------ |
+|              | `--embed-runtime-image-name=NAME`    |  The image name and tag of a container to include in the package. Only for provider and function packages. |
+|              | `--embed-runtime-image-tarball=PATH` |  The filename of a container to include in the package. Only for provider and function packages.                              |
+| `-e`         | `--examples-root="./examples"`       |  The path to a directory of examples related to the package.                               |
+|              | `--ignore=PATH,...`                  |  List of files and directories to ignore.                              |
+| `-o`         | `--package-file=PATH`                |  Directory and filename of the created package.                             |
+| `-f`         | `--package-root="."`                 |  Directory to search for YAML files.                              |
+{{< /table >}}
+
+The `crossplane xpkg build` command recursively looks in the directory set by 
+`--package-root` and attempts to combine any files ending in `.yml` or `.yaml` 
+into a package.
+
+All YAML files must be valid Kubernetes manifests with `apiVersion`, `kind`, 
+`metadata` and `spec` fields. 
+
+#### Ignore files
+
+Use `--ignore` to provide a list of files and directories to ignore.  
+
+For example,  
+`crossplane xpkg build --ignore="./test/*,kind-config.yaml"`
+
+#### Set the package name
+
+`crossplane` automatically names the new package a combination of the 
+`metadata.name` and a hash of the package contents and saves the contents 
+in the same location as `--package-root`. Define a specific location and 
+filename with `--package-file` or `-o`.  
+
+For example,  
+`crossplane xpkg build -o /home/crossplane/example.xpkg`.
+
+
+#### Include examples
+
+Include YAML files demonstrating how to use the package with `--examples-root`. 
+
+[Upbound Marketplace](https://marketplace.upbound.io/) uses files included with 
+`--examples-root` as documentation for published packages.
+
+#### Include a runtime image
+
+Functions and Providers require YAML files describing their dependencies and 
+settings as well as a container image for the function or provider controller.
```

Comment:
This could be read as if a function has a controller. It doesn't - that's why we say runtime rather than controller in the flag.

Reply:
Is there another word besides "runtime" I could use? Would "container image for the function or provider binary" work?

Reply:
It would. I'd like to get folks associating "runtime" as meaning that in this context, so I'd prefer to keep that word in there somewhere to make the connection.

Reply:
Good enough reason for me. I'll change it.